src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -549,7 +555,139 @@ public static String composeRegularCallableMethodHeaderSampleCode(
 
     List<Statement> bodyStatements = new ArrayList<>();
 
-    if (!method.isPaged()) {
+    if (method.isPaged()) {
```

Comment:
This is a very large block, can we extract it into a separate method?

Reply:
Updated.